torch/_torch_docs.py

```diff
@@ -2336,6 +2338,12 @@ def merge_dicts(*dicts):
     tensor([[ 1.9314,  1.2251, -0.0889],
             [ 1.2251,  2.4439,  0.2122],
             [-0.0889,  0.2122,  0.1412]])
+    >>> a = torch.randn(3, 2, 2)
+    >>> a = a @ a.mT + 1e-03 # make symmetric positive-definite
+    >>> l = torch.cholesky(a)
```

Comment:
Let's update this to use the non-deprecate linalg operators (so `linalg.cholesky` here and above and `linalg.inv` instead of `inverse`)

Reply:
I reflected it.
@mruberry Is it possible that functions like torch.cholesky_inverse will be deprecated in near future?

Reply:
We intend to port all the linear algebra functions to `torch.linalg`, so I'd say yes. Now, "the near future" might very well span from 6 months to a full year.

Reply:
Great! I hope to contribute in the process if I can afford to do so at that time. :)
For now, I will just revise the docs.

Reply:
nit: Seems like this isn't updated to `torch.linalg.cholesky` yet.